index.html

```diff
@@ -0,0 +1,365 @@
+         </div>
+        <div class="lets_contact">
+          <div class="contact">
+            <h2>LET’S CONNECT</h2>
+        </div>
+       
+            <section class="social_media">
+                <ul class="social-icons-set">
+                  <li class="social-icon">
+                    <a href="#"><i class="fa-brands fa-twitter" target="_blank" aria-label="Twitter"></i></a>
+                  </li>
+                  <li class="social-icon">
+                    <a href="#"><i class="fa-brands fa-linkedin-in" target="_blank" aria-label="Linkedin"></i></a>
+                  </li>
+                  <li class="social-icon"><a href="#"><i class="fa-solid fa-m" target="_blank"
+                        aria-label="Medium"></i></a>
+                  </li>
+                  <li class="social-icon">
+                    <a href="#"><i class="fa-brands fa-github" target="_blank" aria-label="GitHub"></i></a>
+                  </li>
+                  <li class="social-icon">
+                    <a href="#"><i class="fa-brands fa-angellist" target="_blank" aria-label="Angellist"></i></a>
+                  </li>
+                </ul>
+              </section>
+            </section>
+          </section>
+    </div>
+
+    <!-- start works section -->
+
+    <section class="work_section">    
+
+        <!-- Start first card -->
+
+        <div class="middle_section">
+            <div class="pic">
+                <img src="Snapshoot Portfolio.png" alt="this is a pic">
+            </div>
+            <div class="name">
+                <h1>
+                    Mahram Hossain
+                </h1>
+            </div>
+            <div class="descript flex">
+                <div class="camp">
+                    <h5>
+                        CANOPY
+                    </h5>
+                </div>
+                <div class="year">
+                    <ul class="list flex">
+                        <li>Back And Dev</li>
+                        <li>2015</li>
+                    </ul>
+                </div>
+            </div>
+            <div class="task">
+                <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+            </div>
+            <div class="web flex">
+                <div class="right flex">
+                    <ul>
+                        <li>
+                            <a href="#">HTML</a>
+                        </li>
+                        <li>
+                            <a href="#">CSS</a>
+                        </li>
+                        <li>
+                            <a href="#">JavaScript</a>
+                        </li>
+                    </ul>
+                </div>
+                <div class="left"></div>
+            </div>
```

Comment:
[OPTIONAL] I think it's better to use ``space-between`` because ``space-around`` leaves padding on the left, which is what you don't want 😉 

![suggest](https://user-images.githubusercontent.com/102636489/228645111-72d801da-094d-4a47-ab64-54d152f74499.png)